### DISCUSSION TITLE
Waypoint Spawn Order

### DISCUSSION THREAD

**Discussion by robertrkessler** [mozilla/hubs#2931] [General Questions & Discussion] [answered]
Is there a way to specify a spawn order for waypoints, or is it completely random? Thanks!

**Comment by emclaren** [comment, marked as answer]
Hi Robert, by default visitors spawn randomly into the unoccupied spawnpoints in the room. 

If you want people to load into  a specific spawn point, you can now add an anchor into the URL for the spawnpoints in your scene - more info here: https://github.com/mozilla/hubs/pull/2837

**Comment by robertrkessler** [comment]
Wow. That’s cool. Thanks!!

B.
